src/main/resources/runners/sigtest/pom.xml

```diff
@@ -139,7 +106,9 @@
                   </execution>
                 </executions>
                 <configuration>
+                  <sigfile>${project.build.directory}/sigtest-copy/sigtest/sigtest-1.6-batch.standalone.tck.sig-2.1-se11-OpenJDK-J9</sigfile>
                   <action>strictcheck</action>
+                  <release>11</release>
```

Comment:
This is the Java version, e.g. Java SE 11. The plugin will use this to find the appropriate signatures to use from the `$JAVA_HOME/lib/ct.sym` file. This is effectively a ZIP archive where they store signature files.

Reply:
@jamezp How does this configuration work with other different Java versions like 17 and 21? It does work under JDK 21 locally in my env.

Reply:
It looks in that `ct.sym` archive and finds the Java 11 modules `*.sig` file and uses those. Those are part of the JDK and used in the `--release` argument to compile with the correct signature, or something like that :) Essentially, it's like testing the signature against Java 11 which is all you really need.

Reply:
@jamezp With the above change, this configuration also works under my local JDK21 env, so I wonder how it passes the tests.

Reply:
It's because it uses the Java 11 signatures in the `ct.sym` file that is in your Java 21 home directory under the `lib` directory. It doesn't attempt to check against Java 21 signatures because it shouldn't as the signatures may be different under Java 21, however they still work.

Reply:
Now I understand the details :D Thanks for teaching James!